configure.in

```diff
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_PREREQ(2.58)
```

Comment:
Why this large a version jump? Autoconf 2.71 was only released 2 years ago and isn't in linux distros like Centos Stream 9 or Debian 12. wxWidgets tends to be conservative in its support for tools (hence also only supporting up to C++11, and that was only increased very recently).

Reply:
I thought 2 years was plenty of time :-).  Debian 11 is now oldstable, replaced by Debian 12 Bookworm, which has 2.71.  Also, even if Autoconf 2.71 is not available on some old platform, the configure script should work the same (and it's already distributed in the source archive of wxWidgets), so unless someone wants to modify the wxWidgets Autotools-based build system on oldstable or Centos 9, that shouldn't be an issue.  What do you think?